content/en/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice.md

```diff
@@ -36,4 +36,4 @@ Many open source frameworks and runtimes implement and support MicroProfile Conf
 <!-- lessoncontent -->
 
 ## Example: Externalizing config using MicroProfile, ConfigMaps and Secrets
-### [Start Interactive Tutorial](/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/) 
\ No newline at end of file
+### [Start Interactive Tutorial](/docs/tutorials/configuration/configure-java-microservice/configure-java-microservice-interactive/) 
```

Comment:
What kind of change you perform in there? Since there's nothing changed perhaps we can leave it as it is 🤔

Reply:
Unintentional change. I initiated the edit using browser and it added \n that I didn't notice. Will remove it.